src/components/Icon/index.js

```diff
@@ -237,6 +437,22 @@ const ICONS = {
 
 export const icons = Object.keys(ICONS)
 
+const smallViewBoxIcons = [
```

Comment:
Not related to this PR but I wonder if there would be a better way to handle svg than inlining them like this :)

Reply:
IDK, `React.lazy(() => import('./icons/myicon.svg'))` and wrap it in a suspense may be ? I really don't have good idea because there is no many relevant way to do this 😅